### PR TITLE
Estimates wording

### PR DESCRIPTION
**Pull request by CSavi**
This PR includes:
- shows estimate label on step 1 when estimations are calculated (including estimations from cloned data providers
- refactoring estimates logic and moving most of the data manipulation logic inside the EstimateContainer component
- modifying the store object for ExportInfo's provider estimates (now providerInfo)
- cleaning up and adding tests
- revising few styles for step 1 to accommodate the estimate label
- adding loading state when estimates are being calculated 
- adding edge case scenarios if no estimates are found and if getting estimates

### CODE REVIEW

File: eventkit_cloud/api/views.py [hunk 1]
```diff
@@ -862,6 +862,10 @@ def status(self, request, slug=None, *args, **kwargs):
 
         except Exception as e:
             logger.error(e)
+            import traceback
```
Comment (Caboose71) [inline eventkit_cloud/api/views.py]: This traceback needs to be removed

File: eventkit_cloud/api/views.py [hunk 2]
```diff
@@ -1986,7 +1990,7 @@ def get(self, request, *args, **kwargs):
          Args:
              slugs: Comma separated list of slugs for provider slugs (e.g. 'osm,some_wms1')
              bbox: Bounding box as w,s,e,n (e.g. '-130,-45,-100,10)
-             srs: EPSG code for the bbox srs (default=4326)
+             srqus: EPSG code for the bbox srs (default=4326)
```
Comment (Caboose71) [inline eventkit_cloud/api/views.py]: seems like an accidental typo addition?

File: eventkit_cloud/ui/static/ui/app/components/StatusDownloadPage/DataPackOptions.tsx [hunk 3]
```diff
@@ -95,13 +98,19 @@ export class DataPackOptions extends React.Component<Props, State> {
                     maxZoom: (dataProviderTask) ? dataProviderTask.max_zoom : null,
                     formats: (dataProviderTask) ? dataProviderTask.formats : null,
                 } as Eventkit.Store.ProviderExportOptions;
+
+                // providerInfo[providerTask.slug] = {
```
Comment (Caboose71) [inline eventkit_cloud/ui/static/ui/app/components/StatusDownloadPage/DataPackOptions.tsx]: remove commented out code
